### DISCUSSION TITLE
Fresh Installation on IDH With ERROR 5000 Certificate Signed by Unknown Authority

### DISCUSSION THREAD

**Discussion by xuanhieu0812** [Security-Onion-Solutions/securityonion#10252] [Unsupported Versions]
I have a fresh installation IDH 2.3.220 incompletely. I have trouble with the [ERROR   ] Failed to pull homsn01:5000/security-onion-solutions/so-soc:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")

My hostname managersearch is homsn01. The all services are missing with the same error when I show 'sudo so-status'.

so-filebeat ---------------------------------------------------------------------------------------- [ MISSING ]
    so-idh --------------------------------------------------------------------------------------------- [ MISSING ]
    so-sensoroni --------------------------------------------------------------------------------------- [ MISSING ]
    so-telegraf ---------------------------------------------------------------------------------------- [ MISSING ]
    so-wazuh ------------------------------------------------------------------------------------------- [ MISSING ]

Logs in /root/sosetup.log:

----------
          ID: so-idh
    Function: docker_container.running
      Result: False
     Comment: Failed to pull homsn01:5000/security-onion-solutions/so-idh:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")
     Started: 06:47:45.173525
    Duration: 69.204 ms
     Changes:
----------

----------
          ID: so-filebeat
    Function: docker_container.running
      Result: False
     Comment: Failed to pull homsn01:5000/security-onion-solutions/so-filebeat:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")
     Started: 09:18:08.587199
    Duration: 58.704 ms
     Changes:
----------

----------
          ID: so-wazuh
    Function: docker_container.running
      Result: False
     Comment: Failed to pull homsn01:5000/security-onion-solutions/so-wazuh:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")
     Started: 09:12:29.956499
    Duration: 70.897 ms
     Changes:
----------

----------
          ID: so-telegraf
    Function: docker_container.running
      Result: False
     Comment: Failed to pull homsn01:5000/security-onion-solutions/so-telegraf:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")
     Started: 09:12:27.401542
    Duration: 73.693 ms
     Changes:
----------

----------
          ID: so-sensoroni
    Function: docker_container.running
      Result: False
     Comment: Failed to pull homsn01:5000/security-onion-solutions/so-soc:2.3.220: Error 500: Get https://homsn01:5000/v2/: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "homsn01")
     Started: 09:12:26.005094
    Duration: 64.101 ms
     Changes:
----------

**Comment by InfosecGoon** [comment]
Do you have any sort of SSL interception or proxy between these nodes?

**Comment by xuanhieu0812** [comment]
I set network direct to manager. I have re-installed IDH and got the problem about cert

**Comment by xuanhieu0812** [comment]
All firewalls are already opened between manager and IDH to communicate
